### PR TITLE
Fixes and documentation for compas_fab.robots.Robot.basic method

### PR DESCRIPTION
**Pull request by tetov**
### What type of change is this?

- [x] Bug fix in a **backwards-compatible** manner.

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.rst` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`) .
- ~~[ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.Configuration`.~~
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_fab/robots/robot.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def __init__(self, model, artist=None, semantics=None, client=None):
         self._scale_factor = 1.
         self.model = model
         self.attached_tool = None
-        self.artist = artist  # setter and getter (because of scale)
+        self._artist = artist  # setter and getter (because of scale)
```
Comment (tetov) [inline src/compas_fab/robots/robot.py]: No need to set `self.artist` until it's referenced.

Comment (gonzalocasas) [reply src/compas_fab/robots/robot.py]: This doesn't seem right; the `artist` setter does a few things related to scaling and what not, if you skip it, when does that happen?

File: src/compas_fab/robots/robot.py [hunk 2]
```diff
@@ -64,23 +64,37 @@ def artist(self):
...
+        """Convenience method to create the most basic instance of a robot,
+           based only on a name.
 
         Parameters
         ----------
         name : str
             Name of the robot
+        joints : :class:`compas.robots.Joint`, optional
+        links : :class:`compas.robots.Link`, optional
+        materials : `compas.robots.Material`, optional
```
Comment (beverlylytle) [inline src/compas_fab/robots/robot.py]: Missing `:class:`

File: src/compas_fab/robots/robot.py [hunk 3]
```diff
@@ -64,23 +64,37 @@ def artist(self):
...
+        **kwargs
+            Keyword arguments passed to :class:`compas.robots.RobotModel`
+            accessible from `cls.model.attr`
```
Comment (beverlylytle) [inline src/compas_fab/robots/robot.py]: Missing `:attr:`

File: tests/robots/test_robot.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+@pytest.fixture
+def ur5_links():
+    robot = Ur5Robot()
+    return robot.model.links
+
+
+def test_basic_name_only():
+    robot = Robot.basic('testbot')
+    print('hi')
```
Comment (beverlylytle) [inline tests/robots/test_robot.py]: Hi!

Comment (tetov) [reply tests/robots/test_robot.py]: Hi and thanks for your review :)

File: src/compas_fab/robots/robot.py (hunk 3 above)
Comment (tetov) [reply src/compas_fab/robots/robot.py]: Did not manage to get links for attr.

File: CHANGELOG.rst [hunk 5]
```diff
@@ -17,6 +17,9 @@ Unreleased
 
 **Changed**
 
+* Property `compas_fab.robots.Robot.artist` does not try to scale robot
```
Comment (beverlylytle) [inline CHANGELOG.rst]: Maybe this also deserves a :class: prefix...

Comment (tetov) [reply CHANGELOG.rst]: It works nicely on the docs website but not in the Github preview, see [my fork](https://github.com/tetov/compas_fab/blob/allow_skeleton_robot_instances/CHANGELOG.rst).

**Pull request merged**